app/models/miq_worker.rb

```diff
@@ -317,16 +317,36 @@ def send_message_to_worker_monitor(message, *args)
     )
   end
 
+  def self.preload
+  end
+
+  def self.pre_fork
+    preload
+  end
+
+  def self.after_fork
+    ActiveRecord::Base.connection.reconnect!
+    DRb.stop_service
+  end
+
   def start
-    msg = "Worker started: ID [#{id}], PID [#{pid}], GUID [#{guid}]"
-    MiqEvent.raise_evm_event_queue(miq_server, "evm_worker_start", :event_details => msg, :type => self.class.name)
+    self.class.pre_fork
+    pid = fork do
+      self.class.after_fork
+      self.class::Runner.start_worker(command_line_params)
+      exit!
+    end
 
-    ENV['MIQ_GUID'] = guid
-    self.pid = Kernel.spawn(command_line, :out => "/dev/null", :err => [Rails.root.join("log", "evm.log"), "a"])
     Process.detach(pid)
+    ActiveRecord::Base.connection.reconnect!
```

Comment:
I think it stemmed from https://github.com/grosser/parallel/issues/62, https://github.com/grosser/parallel/issues/83, and rails 3.2/4.0 in https://github.com/rails/rails/issues/12867.  I have removed the `reconnect!` in both the parent/child post-fork locally and everything started fine.  I'm leaning towards removing the `reconnect!` since maybe @matthewd fixed the connection pool issue(s) that required the `reconnect!` in the child/parent post-fork.